src/auth/session/storage/custom.ts

```diff
@@ -5,20 +5,28 @@ import * as ShopifyErrors from '../../../error';
 
 export class CustomSessionStorage implements SessionStorage {
   constructor(
-    readonly storeCallback: (session: SessionInterface) => Promise<boolean>,
-    readonly loadCallback: (
+    readonly storeSessionCallback: (
```

Comment:
Yeah, it's a fair point about it potentially be breaking.  In theory, they _shouldn't_ call `myStorage.storeCallback`, since the goal is to have something that responds to `storeSession` by the library when needed, but ... 

I thought it would be nice to be consistent but I'll change them back to avoid problems.